code/modules/mob/living/carbon/alien/humanoid/caste/hunter.dm

```diff
@@ -12,9 +12,9 @@
 	alien_organs += new /obj/item/organ/internal/xenos/plasmavessel/hunter
 	..()
 
-/mob/living/carbon/alien/humanoid/hunter/movement_delay()
-	. = -1		//hunters are sanic
-	. += ..()	//but they still need to slow down on stun
+///mob/living/carbon/alien/humanoid/hunter/movement_delay() //May be re-enabled later with less harsh values.
```

Comment:
Don't leave commented code in the repo. It's a messy practice and all the history is stored in git if you want to get it later.

Reply:
Oh, I wasn't aware history was stored in the git. I'm very unfamiliar with github, so I'll keep this in mind for the future. Thanks!